src/components/Records/SearchFilters.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="container">
+  <div class="container" id="advancedSearch">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['id' should be on a new line.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=5059186)